frontend/src/routes/portfolio/+page.svelte

```diff
@@ -0,0 +1,52 @@
+<div class="block mt-5 ml-5">
+    <h1 class="title is-2">Portfolio</h1>
+</div>
+
+<div class="columns mt-5">
+    <div class="column is-two-fifths">
+        <h2 class="title is-5 ml-5">Summary</h2>
+        <div class="block mx-5 has-background-grey-darker">
```

Comment:
Using `box` instead of `block` here gives a nicer effect.

Reply:
👌